### PR TITLE
Reduce verbose logging

### PR DESCRIPTION
**Pull request by lzr-official**
Reduce verbose logging for `grpc_channel.cc` by changing one of `LOG(INFO)` to `VLOG(2)`.

### CODE REVIEW

**Review by rohan100jain — APPROVED** [review]

**Review by haoyuz — APPROVED** [review]

File: tensorflow/core/distributed_runtime/rpc/grpc_channel.cc [hunk 1]
```diff
@@ -269,7 +269,7 @@ class SparseGrpcChannelCache : public CachingGrpcChannelCache {
         job_id_(job_id),
         host_ports_(host_ports),
         channel_func_(std::move(channel_func)) {
-    LOG(INFO) << "Initialize GrpcChannelCache for job " << ToString();
+    VLOG(2) << "Initialize GrpcChannelCache for job " << ToString();
```
Comment (mihaimaruseac) [inline tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:272]: Why not use `LOG(DEBUG)` instead to keep using the same factory of functions?

(driving by review as I notice this PR has issues importing into google codebase)

Comment (mihaimaruseac) [reply tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:272]: (ignore the issues part, seems to be resolved now)

Comment (lzr-official) [reply tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:272]: Seems like all other logs are VLOG in this file. Would it be appropriate to use VLOG or LOG(DEBUG) from your point of view?

Comment (mihaimaruseac) [reply tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:272]: Oh, in that case for consistency, please use `VLOG`. I didn't realize that.

**Pull request merged**